doc/api/util.md

```diff
@@ -71,6 +71,25 @@ callbackFunction((err, ret) => {
 });
 ```
 
+## `util.colorText(format, text)`
+
+<!-- YAML
+added: v18.3.0
+-->
+
+* `format` {string} `format` one of the color format from `util.inspect.colors`
+* `text` {string} The text you would like to color
+* Returns: {string} colored text string
+
+Takes `format` and `text` and retuns the colored text form
+
+```js
+const util = require('node:util');
```

Comment:
Can you remove this line. Otherwise we'll need to show the same example code for CJS and ESM.